README.md

```diff
@@ -17,138 +19,143 @@ Before getting started with this project, please get yourself familiar with the
+```shell
+cd $proj_root/iot-hub-device-update/yocto 
+```
+3. Once you're in the `yocto` directory we need to check out the Yocto Build "engine" or base layer so we can build with it. 
+```shell
+git clone --depth 1 --branch $yocto_release git://git.yoctoproject.org/poky
+```
+4. Next we need to checkout the rest of the dependency layers into the `yocto` directory
 
-- Clone SWUpdate meta layer. SWUpdate provides an image-based update that support dual-partition.
-  
+    1. Clone the SwUpdate meta layer 
     ```shell
-    # Clone swupdate meta layer 
     git clone --depth 1 --branch $yocto_release  https://github.com/sbabic/meta-swupdate
     ```
 
-- Clone the Open Embedded meta layer. This layer include many modules (or layers) needed for building a Linux-base system.
-
+    2. Clone the Open Embedded meta layer. This layer include many modules (or layers) needed for building a Linux-base system.
     ```shell
     git clone --depth 1 --branch $yocto_release  git://git.openembedded.org/meta-openembedded
     ```
 
-- Clone the Raspberry Pi meta layer. Since, the reference image that we are building is for a Raspberry Pi 3B+ hardware.
-
+    3. Clone the Raspberry Pi meta layer. Since, the reference image that we are building is for a Raspberry Pi 4 hardware.
     ```shell
     git clone --depth 1 --branch $yocto_release git://git.yoctoproject.org/meta-raspberrypi
     ```
 
-- Clone the Azure Device Update meta layer.
+5. Within the same directory we are now going to include the Device Update for IotHub layers which builds Device Update and provides those artifacts for the `meta-raspberrypi-adu` layer which integrates Device Update and modifies the image build instructions within `meta-raspberrypi` to output the `adu-base-image-<machine-name>.wic.gz` and `adu-update-image-<machine-name>.swu` which can be used to test out Device Update for IotHub. For more information on these layers and their outputs please read the `README.md` in each of the repos. 
+
+    1. From within the `yocto` directory checkout `meta-azure-device-update` at the version of Device Update you plan to use in your test. 
 
     ```shell
     git clone --branch $adu_release http://github.com/azure/meta-azure-device-update
     ```
-
-- Clone the IoT Hub Device Update Delta meta layer.
-
+    2. From within the `yocto` directory checkout `meta-raspberrypi-adu` at the version of Device Update you plan to use in your test. 
     ```shell
-    git clone --branch $adu_release http://github.com/azure/meta-iot-hub-device-update-delta
+    git clone --branch $adu_release http://github.com/azure/meta-raspberrypi-adu
     ```
-
-- Clone the Raspberry Pi with ADU meta layer.
-  
+    3. (optional) If you're planning on using delta updates you can checkout the `meta-iot-hub-device-update-delta` layer that integrates that functionality into that agent. Please read the `README.md` if you want to know more. 
```

Comment:
Is it not appropriate in this context to reference learn.ms.com documentation? I think the ideal starting point for learning about ADU delta updates is: https://learn.microsoft.com/azure/iot-hub-device-update/delta-updates. But maybe the context here is to stay within the GitHub repo?

Reply:
Good call. Are there other areas where we could leverage the learn.ms docs? I think maybe linking to the guide for importing an update might be useful as well. 

The audience for these is intended to be developers who have some experience with building and working with embedded devices but I we can lean on our other documentation to bridge the gap and reach a wider audience.